Makefile

```diff
@@ -1,5 +1,5 @@
 
-export KUBEVIRTCI_TAG ?= $(shell curl -L https://storage.googleapis.com/kubevirt-prow/release/kubevirt/kubevirtci/latest)
+export KUBEVIRTCI_TAG ?= $(shell curl -L -Ss https://storage.googleapis.com/kubevirt-prow/release/kubevirt/kubevirtci/latest)
```

Comment:
maybe one time we do want to print it , in the provider.sh ?

Reply:
Its will print each iteration of the progress bar in new line, its a nosi in the logs that I want to reduce.
Specificity `-S` will print the result of the request to download form the URL so we get some feedback about download status.

Reply:
I am fine with reducing the noise,
i think it wont print the TAG at all (tried here, unless i am wrong), this is what i would suggest to add please one print in provider.sh
`echo $KUBEVIRTCI_TAG`

Reply:
Sorry I though you talk about downloads in general
At the current state it dosent print KUBEVIRT_TAG at all the flags that I added dont change it, it affects only the download progress messages.
If we want to decide to print the tag we need to do it for all providers so it will be consistent,
I think it should be done on another PR

Reply:
Sorry I though you talk about downloads in general.
At the current state it dosent print KUBEVIRT_TAG at all the flags that I added dont change it, it affects only the download progress messages.
If we want to decide to print the tag we need to do it for all providers so it will be consistent,
I think it should be done on another PR

Reply:
adding it to `provider.sh` of SRIOV wont affect any other lane
but up to you

Reply:
If something going wrong that you would want to check the code on a sertian tag
its probably something that happen on kubevirt.
And when you run one of the lanes KUBEVIRTCI_TAG is saved at cluster-up/hack/common.sh
which is shown in the job logs, so you could find on which tag there was a problem.

Printing KUBEVIRTCI_TAG  should be on `make cluster-up` IMO and not only on one provider